cattle/plugins/cadvisor/cadvisor.py

```diff
@@ -10,12 +10,17 @@ def on_startup(self):
         cmd = ['cadvisor',
                '-logtostderr=true',
                '-listen_ip', Config.cadvisor_ip(),
-               '-port', str(Config.cadvisor_port())]
+               '-port', str(Config.cadvisor_port()),
+               '-housekeeping_interval', Config.cadvisor_interval()]
 
         docker_root = Config.cadvisor_docker_root()
         if docker_root:
             cmd += ["-docker_root", docker_root]
 
+        cadvisor_opts = Config.cadvisor_opts()
```

Comment:
I think this string needs to be parsed by some `shlex` library.  For example, if `CADVISOR_OPTS=-a b -c d`, it seems like that whole string would be passed as one argument and not 4 individual arguments.